addons/iot_drivers/iot_system.py

```diff
@@ -0,0 +1,16 @@
+
+from enum import Enum, unique
+import platform
+
+
+@unique
+class IoTSystem(Enum):
+    RASPBERRY_PI = 'L'
+    WINDOWS = 'W'
+
+
+IOT_SYSTEM = IoTSystem.WINDOWS if platform.system() == 'Windows' else IoTSystem.RASPBERRY_PI
+"""IoT system type detected for the current environment"""
+
+IS_RPI = IOT_SYSTEM == IoTSystem.RASPBERRY_PI
+IS_WINDOWS = IOT_SYSTEM == IoTSystem.WINDOWS
```

Comment:
Couldn't this be simplified by

```suggestion
import platform

IOT_SYSTEM = platform.system()
"""IoT system type detected for the current environment"""

IS_RPI = IOT_SYSTEM == "Linux"
IS_WINDOWS = IOT_SYSTEM == "Windows"
```

Reply:
I was also using the char in the "load_handler" part. I used something simpler in the new version, let me know what you think of it :)

Reply:
yeah it looks better, but I think it's still easier to read if we just use `IOT_SYSTEM[0]` when we need to get the letter, and `IOT_SYSTEM[0] == "L"` every time we want to check if the letter corresponds to Linux. 

I mean, for someone new or even for us, we can know what is an IoT System, but not really and IOT_SYSTEM_CHAR, and even if we knew, it's used only a very small amount of times right ?